public/documentation/_data.json

```diff
@@ -13,6 +13,9 @@
     "title": "开始",
     "origin": "getting_started"
   },
+  "crosswalk-app-tools": {
```

Comment:
Are we sure that put the crosswalk-app-tools into the left navigation bar for english verison?